hybridse/src/proto/fe_common.proto

```diff
@@ -22,37 +22,81 @@ option java_outer_classname = "Common";
 enum StatusCode {
     kOk = 0;
     kRunning = 1;
+    // General error
     kRunError = 2;
-    kTypeError = 8;
-    kFileIOError = 9;
-    kUnSupport = 10;
-    kBadRequest = 11;
-    kNameExists = 12;
-    kTableExists = 13;
-    kNoDatabase = 14;
-    kTableNotFound = 15;
-    kTablePutFailed = 16;
-    kNullPointer = 17;
-    kProcedureNotFound = 18;
-    kConnError = 20;
-    kRpcError = 21;
-    kTimeoutError = 22;
-    kResponseError = 23;
-    kSyntaxError = 29;
-    kSqlError = 30;
-    kPlanError = 31;
-    kCallMethodError = 32;
-    kColumnNotFound = 33;
-    kIndexNotFound=34;
-    kFunError = 40;
-    kCodegenError = 50;
-    kOpGenError = 60;
-    kJitError = 70;
-    kSchemaCodecError = 71;
+    kTypeError = 3;
```

Comment:
add some brief comment ?

Reply:
Or maybe we can make the enum string much more understandable.